src/components/Notification/StatusMessage.jsx

```diff
@@ -18,19 +24,35 @@ import React from 'react';
 const StatusMessage = ({ notification, locationUrl }) => {
   if (locationUrl) {
     return (
-      <dd className="labelStatus" role="alert">
-        {notification}{' '}
-        <a href={locationUrl} target="_blank" rel="noreferrer">
-          {locationUrl}
+      <Typography
+        variant="dd"
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center'
+        }}
+      >
+        {notification}
+        <a href={locationUrl} target="_blank" rel="noreferrer" style={{ marginLeft: 3 }}>
+          Click here.
         </a>
-      </dd>
+      </Typography>
     );
   }
 
   return (
-    <dd className="labelStatus" role="alert">
+    <Typography
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center'
+      }}
```

Comment:
Lines 45 to 50 can be removed. Styling for centering the text is already handled by StatusNotification via `<Box>`.